r-package-manager-demo/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
You probably want to leave this at the root

Reply:
Do we also want the LICENSE included with the packages (e.g., the source, binary, and Python distributions we serve from solo)?

Reply:
Oh interesting, I didn't think about the Package Manager side of things... these make sense